ppr-ui/tests/unit/ButtonFooter.spec.ts

```diff
@@ -378,3 +381,125 @@ describe('Button events', () => {
     expect(getLastEvent(wrapper, 'error')).not.toBeNull()
   })
 })
+
+describe('Mhr Manufactuer Registration step 1 - Your Home', () => {
+  let wrapper: Wrapper<any>
+  let wrapper2: any // eslint-disable-line no-unused-vars
+  const currentStatementType = StatementTypes.FINANCING_STATEMENT
+  const currentStepName = RouteNames.YOUR_HOME
+
+  beforeAll(async () => {
+    if (router.currentRoute.name !== RouteNames.YOUR_HOME) {
+      router.replace({ name: RouteNames.YOUR_HOME })
+    }
+    await store.setAuthRoles(mockedManufactuerAuthRoles)
+    await store.setRegistrationType(MhrRegistrationType)
+  })
+
+  afterAll(async () => {
+    await store.setAuthRoles([])
+    await store.setRegistrationType(null)
+  })
+
+  beforeEach(async () => {
+    const localVue = createLocalVue()
+    localVue.use(VueRouter)
+    wrapper2 = shallowMount((YourHome as any), { localVue, store, router, vuetify })
```

Comment:
Do we need wrapper2?

Reply:
I was doing it based on the previous tests. But I don't believe its needed. Ill remove it.